### PR TITLE
fix: Convert HTML to MD for docs/index.md

### PR DESCRIPTION
**Pull request by nschonni**
Also a small on in the latest release log

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: locale/zh-cn/download/releases.md [hunk 1]
```diff
@@ -8,14 +8,15 @@ modules: "<code>NODE_MODULE_VERSION</code> 指的是 Node.js 的 ABI (applicatio
...
+
+#### 正在寻找某个大版本的最新版？
+
+* [Node.js 10.x](https://nodejs.org/dist/latest-v10.x/)
+* [Node.js 8.x](https://nodejs.org/dist/latest-v8.x/)
+* [Node.js 6.x](https://nodejs.org/dist/latest-v6.x/)
+* [Node.js 4.x](https://nodejs.org/dist/latest-v4.x/)
+* [Node.js 0.12.x](https://nodejs.org/dist/latest-v0.12.x/)
+* [Node.js 0.10.x](https://nodejs.org/dist/latest-v0.10.x/)
+* [all versions](https://nodejs.org/dist/)
```
Comment (amorist) [inline locale/zh-cn/download/releases.md]:
```suggestion
* [所有版本](https://nodejs.org/dist/)
```

File: locale/zh-tw/download/releases.md [hunk 2]
```diff
@@ -8,14 +8,15 @@ modules: "<code>NODE_MODULE_VERSION</code> 是 Node.js ABI (application binary i
...
+
+#### 在找某個主要版本的最新版嗎？
+
+* [Node.js 10.x](https://nodejs.org/dist/latest-v10.x/)
+* [Node.js 8.x](https://nodejs.org/dist/latest-v8.x/)
+* [Node.js 6.x](https://nodejs.org/dist/latest-v6.x/)
+* [Node.js 4.x](https://nodejs.org/dist/latest-v4.x/)
+* [Node.js 0.12.x](https://nodejs.org/dist/latest-v0.12.x/)
+* [Node.js 0.10.x](https://nodejs.org/dist/latest-v0.10.x/)
+* [all versions](https://nodejs.org/dist/)
```
Comment (amorist) [inline locale/zh-tw/download/releases.md]:
```suggestion
* [所有版本](https://nodejs.org/dist/)
```

File: locale/zh-cn/docs/index.md [hunk 3]
```diff
@@ -20,19 +20,19 @@ labels:
...
+
+* [Node.js 9.x](https://nodejs.org/docs/latest-v9.x/api/)
+* [Node.js 8.x](https://nodejs.org/docs/latest-v8.x/api/)
+* [Node.js 7.x](https://nodejs.org/docs/latest-v7.x/api/)
+* [Node.js 6.x](https://nodejs.org/docs/latest-v6.x/api/)
+* [Node.js 5.x](https://nodejs.org/docs/latest-v5.x/api/)
+* [Node.js 4.x](https://nodejs.org/docs/latest-v4.x/api/)
+* [Node.js 0.12.x](https://nodejs.org/docs/latest-v0.12.x/api/)
+* [Node.js 0.10.x](https://nodejs.org/docs/latest-v0.10.x/api/)
+* [All versions](https://nodejs.org/docs/)
```
Comment (amorist) [inline locale/zh-cn/docs/index.md]:
```suggestion
* [所有版本](https://nodejs.org/docs/)
```

**Comment by nschonni** [comment]
@amorist thanks for catching those! Since I messed that up across the languages, I'm just going to rebase to fix all of them

File: locale/en/docs/index.md [hunk 4]
```diff
@@ -20,19 +20,19 @@ The [API reference documentation](/api/) provides detailed information about a f
...
-    <li><a href="https://nodejs.org/docs/latest-v5.x/api/">Node.js 5.x</a></li>
-    <li><a href="https://nodejs.org/docs/latest-v4.x/api/">Node.js 4.x</a></li>
-    <li><a href="https://nodejs.org/docs/latest-v0.12.x/api/">Node.js 0.12.x</a></li>
-    <li><a href="https://nodejs.org/docs/latest-v0.10.x/api/">Node.js 0.10.x</a></li>
-    <li><a href="https://nodejs.org/docs/">All versions</a></li>
-  </ul>
+
+### Looking for API docs of previous releases?
+
+* [Node.js 9.x](https://nodejs.org/docs/latest-v9.x/api/)
```
Comment (nschonni) [inline locale/en/docs/index.md:26]: Not sure if these should all be pointing at 10 now, or just remove the 9 line

**Review by osk2 — APPROVED** [review]

**Pull request merged**